docker/Dockerfile.olbase

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:xenial
 
+ARG DOCKER_HUB=FALSE
+
```

Comment:
We won't need to set this.  Docker Hub will do it or it will not be set.

Reply:
@cclauss I tried without setting it several times. Turns out you passing an environment variable for the `docker build` command without declaring it in the `Dockerfile` is pretty useless while you **can** pass and **use** the environment variable passed to the `docker run` command. (Took me hours to figure out why wasn't the environment variable getting picked up. :( )

You can also try this using a minimal Dockerfile and building it with an environment variable **not** in the Dockerfile and trying to print the environment variables for the same. The variable is nowhere to be seen. 

**Demo:** In the repo https://github.com/aasifkhan7/test-repo I am building docker image `one` using `docker/Dockerfile.one` and image `two` using `docker/Dockerfile.two` using the same command `docker build -t $IMAGE-NAME --build-arg DOCKER_HUB=TRUE -f docker/$DOCKER-FILE .` (`$IMAGE-NAME` and `$DOCKER-FILE` can be replaced by `one` and `two`) and the outputs are shown in `output.one` and `output.two`

In `output.two`, the environment variable `DOCKER_HUB` is nowhere to be seen, where it is present in `output.one`